stand/efi/Makefile.inc

```diff
@@ -26,6 +26,8 @@ EFI_TARGET=	efi-app-ia32
 .else
 EFI_TARGET=	binary
 .endif
+# XXX: doesn't work with llvm-objdump!
```

Comment:
```suggestion
# XXX: doesn't work with llvm-objcopy!
```
?
(Also, do we know why?)

Reply:
It doesn't like `--output-target=${EFI_TARGET}` (efi-app-x86_64) in this case.